pennylane/templates/layers/particle_conserving_u1.py

```diff
@@ -0,0 +1,251 @@
+
+    The circuit implementing the entangler blocks is shown in the figure below:
+
+    |
+
+    .. figure:: ../../_static/templates/layers/particle_conserving_u1.png
+        :align: center
+        :width: 50%
+        :target: javascript:void(0);
+
+    |
+
+    The repeated units across several qubits are shown in dotted boxes. Each layer
+    contains :math:`N-1` particle-conserving two-parameter exchange gates
+    :math:`U_{1,\mathrm{ex}}(\phi, \theta)` that act on pairs of nearest neighbors qubits.
+    The unitary matrix representing :math:`U_{1,\mathrm{ex}}(\phi, \theta)`
+    is given by (see `arXiv:1805.04340 <https://arxiv.org/abs/1805.04340>`_),
+
+    .. math::
+
+        U_{1, \mathrm{ex}}(\phi, \theta) = \left(\begin{array}{cccc}
+        1 & 0 & 0 & 0 \\
+        0 & \mathrm{cos}(\theta) & e^{i\phi} \mathrm{sin}(\theta) & 0 \\
+        0 & e^{-i\phi} \mathrm{sin}(\theta) & -\mathrm{cos}(\theta) & 0 \\
+        0 & 0 & 0 & 1 \\
+        \end{array}\right).
+
+    The figure below shows the circuit decomposing :math:`U_{1, \mathrm{ex}}` in
+    elementary gates. The Pauli matrix :math:`\sigma_z` and single-qubit rotation :math:`R(0, 2 \theta, 0)`
+    apply the Pauli Z operator and an arbitrary rotation on the qubit ``n``,
+
+    |
+
+    .. figure:: ../../_static/templates/layers/u1_decomposition.png
+        :align: center
+        :width: 80%
+        :target: javascript:void(0);
+
+    |
+
+    :math:`U_A(\phi)` is the unitary matrix
+
+    .. math::
+
+        U_A(\phi) = \left(\begin{array}{cc} 0 & e^{-i\phi} \\ e^{-i\phi} & 0 \\ \end{array}\right),
+
+    acting controlledly on the state of qubit ``m`` which is further decomposed in terms of the
+    `quantum operations <https://pennylane.readthedocs.io/en/stable/introduction/operations.html>`_
+    supported by Pennylane,
+
+    |
+
+    .. figure:: ../../_static/templates/layers/ua_decomposition.png
+        :align: center
+        :width: 70%
+        :target: javascript:void(0);
+
+    |
+
+    where,
+
+    |
+
+    .. figure:: ../../_static/templates/layers/phaseshift_decomposition.png
+        :align: center
+        :width: 65%
+        :target: javascript:void(0);
+
+    |
+
+    The quantum circuits above decomposing the unitaries :math:`U_{1,\mathrm{ex}}(\phi, \theta)`
+    and :math:`U_A(\phi)` are implemented by the :func:`~.u1_ex_gate` and :func:`~.decompose_ua`
+    functions, respectively.
+
+    Args:
+        weights (array[float]): Array of weights of shape ``(D, M, 2)``.
+            ``D`` is the number of entangler block layers and :math:`M=N-1`
+            is the number of exchange gates :math:`U_{1,\mathrm{ex}}` per layer.
+        wires (Iterable or Wires): Wires that the template acts on. Accepts an iterable of numbers
+            or strings, or a Wires object.
+        init_state (array[int]): Length ``len(wires)`` vector representing the Hartree-Fock state.
+        ``init_state`` is used to initialize the wires.
```

Comment:
> ``init_state`` is used to initialize the wires.

This seems to render a bit funky:



![image](https://user-images.githubusercontent.com/24476053/98884254-9c6ed600-245d-11eb-94b4-96593ce6890e.png)


Maybe worth having it in the same line with the def of the `init_state` arg?

Reply:
good catch @antalszava , thanks!